Added option for selective upload.

floydbloke opened the pull request:
Hi LubikR

Nearly a year ago I lodged an issue requesting if a selective upload could be implemented. Recently I decided to have a go at this myself and this is the result.  I modified the Playback activity from the PMCADemo app to create an activity that shows thumbnails and a few details about the image and lets the users select/deselect multiples.  Then made some addtions to the main layout accordingly.
Essentially I added:
*Selection-activity class and activity layout, with supporting classes (ScalingBitmapview and BitmapUtil)
*SelectedImages class - List to hold user selected images.
![20190413_162618 (Large)](https://user-images.githubusercontent.com/26859079/56074590-7ad4e500-5e09-11e9-89b0-feb1390dd1ae.jpg)
![20190413_162556 (Large)](https://user-images.githubusercontent.com/26859079/56074591-80322f80-5e09-11e9-83e2-58686a6b7d06.jpg)

floydbloke committed:
Added option for selective upload.
*Selection-activity class and activity layout, with supporting classes (ScalingBitmapview and BitmapUtil)
*SelectedImages class - List to hold user selected images.

retrography commented:
This is indeed a very nice addition.

LubikR commented:
Hello,
first I have to say: good work! Due to the work on other projects I don't have much time for this one and the functionality I have developed is enough for my needs. However, if someone sends a pull-request, I like to merge it and publish, so once again, thanks for the post.
However, I am currently not able to compile the project, specifically line 34 in the SelectionActivity.java variable R.drawable.checkmarkimage. I was looking into the API and in the R.drawable class I couldn't find int checkmarkimage. Is there a bitmap missing in the pull-request or ...?
Thanks

floydbloke committed:
Added option for selective upload.
*Selection-activity class and activity layout, with supporting classes (ScalingBitmapview and BitmapUtil)
*SelectedImages class - List to hold user selected images.

floydbloke commented:
Thank you.
You are quite correct about the missing bitmap.  Apologies for my oversight.  I've now uploaded the requisite resource file (checkmark.png under res/drawable-hdpi) so that should be in the most recent commit.
(I'm a bit of a rookie at Android Studio and Github.  I added the png file via windows explorer rather than through the IDE and that's apparently why AS didn't include it in the original push.)  Hopefully it all works this time and people find it a useful feature.
